cypress/e2e/yves/ssp-service/ssp-service.cy.ts

```diff
@@ -0,0 +1,178 @@
+
+                isSetupDone = true;
+            }
+
+            sspServiceListPage.visit();
+        };
+
+        afterEach(() => {
+            customerLogoutScenario.execute();
+        });
+
+        describe('Service List Page', () => {
+            it('should display the service list page with table', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+
+                // Assert page is loaded correctly
+                cy.get('h1').should('contain', 'Services');
+                sspServiceListPage.getTable().should('exist');
+            });
+
+            it('should verify all required table headers exist', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+
+                // Check if all column headers are present
+                sspServiceListPage.getTableHeaders().should('have.length.at.least', 5);
+                sspServiceListPage.getTableHeaders().contains('Order Reference').should('exist');
+                sspServiceListPage.getTableHeaders().contains('Service Name').should('exist');
+                sspServiceListPage.getTableHeaders().contains('Time and Date').should('exist');
+                sspServiceListPage.getTableHeaders().contains('Created At').should('exist');
+                sspServiceListPage.getTableHeaders().contains('State').should('exist');
+            });
+
+            it('should sort by Order Reference column in both directions', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+                // Sort by Order Reference
+                sspServiceListPage.clickSortColumn('Order Reference');
+                // Verify that sorting was triggered
+                sspServiceListPage.getOrderByInput().should('have.value', 'order_reference');
+                sspServiceListPage.getOrderDirectionInput().should('have.value', 'ASC');
+
+                // Click again to toggle sort direction
+                sspServiceListPage.clickSortColumn('Order Reference');
+
+                // Verify sort direction was toggled
+                sspServiceListPage.getOrderByInput().should('have.value', 'order_reference');
+                sspServiceListPage.getOrderDirectionInput().should('have.value', 'DESC');
+            });
+
+            it('should sort by Service Name column in both directions', (): void => {
```

Comment:
all 3 'it should sort by' tests can be merged into one. 
Each time we place an order through Yves, it's pretty time-consuming.